Assignment/TestUpdateCount.txt

```diff
@@ -0,0 +1,33 @@
+@isTest
+public class TestUpdateCount
+{
+    @testsetup static void setup()
+    {
+        List<Class__c> classList = new List<Class__c>();
+       
+        
+        for(Integer i=1; i<=10; i++)
+        {
+            Class__c unitClass = new class__c(name = 'class'+ String.valueOf(i), MyCount__c = 0);
+            classList.add(unitClass);         
+        }
+        insert classList;     
+    }
+    
+    @istest static void testP()
```

Comment:
change the test method names to meaningful names so that it can be identified for what it is written